source/server/server.cc

```diff
@@ -470,6 +470,9 @@ void InstanceImpl::initialize(const Options& options,
   // We can now initialize stats for threading.
   stats_store_.initializeThreading(*dispatcher_, thread_local_);
 
+  handler_ =
```

Comment:
I don't know if this causes an actual bug or not, but when `this` gets destructed, the `overload_manager_` will be destructed before `handler_`, so `handler_` will have a dangling reference to a destructed object. Re-order class members?